examples/tensorflow/train_predict.py

```diff
@@ -36,9 +39,12 @@ def main(argv):
         try:
             saved_estimator_path = regressor.export_savedmodel(temp, receiver_fn).decode("utf-8")
             # Logging the saved model
-            tensorflow.log_saved_model(saved_model_dir=saved_estimator_path, signature_def_key="predict", artifact_path="model")
+            mlflow.tensorflow.log_model(tf_saved_model_dir=saved_estimator_path,
+                                        tf_meta_graph_tags = [tag_constants.SERVING],
+                                        tf_signature_def_key="predict",
+                                        artifact_path="model")
             # Reloading the model
-            pyfunc_model = pyfunc.load_pyfunc(saved_estimator_path)
+            pyfunc_model = pyfunc.load_pyfunc(os.path.join(mlflow.get_artifact_uri(),'model'))
```

Comment:
LGTM - could we change this to `mlflow.get_artifact_uri("model"))`

Reply:
@aarondav   I implemented the recommended change